fastlane/Fastfile

```diff
@@ -80,6 +80,37 @@ platform :ios do
     )
   end
 
+  desc "builds and packages iOS apps"
+  lane :archive do
+    # Fetch provisioning profile
+    # sigh
+    sigh(
+      adhoc: false, 
+      username: "hrondon@teclib.com",
+      team_id: "H7XJV96LX2",
```

Comment:
Can you put this team_id and cert_id like a secret variable

Reply:
yes, that's possible

Reply:
changed